packages/commonwealth/client/scripts/views/components/proposals/votingResultCards/YesNoRejectVotingResultCard.tsx

```diff
@@ -0,0 +1,56 @@
+import clsx from 'clsx';
+import React from 'react';
+import { NearSputnikVoteString } from '../../../../controllers/chain/near/sputnik/types';
+import { formatPercent } from '../../../../helpers/index';
+import { CWText } from '../../component_kit/cw_text';
+import { YesNoRejectVotingResultProps } from '../votingResults/YesNoRejectVotingResult';
+import { safeDivision } from './AaveVotingResultCard';
+
+export const YesNoRejectVotingResultCard = (
+  props: YesNoRejectVotingResultProps
+) => {
+  const { votes } = props;
```

Comment:
for future reference if you will be creating new components, please destructure props right away like so
```
export const YesNoRejectVotingResultCard = ({
  votes,
}: YesNoRejectVotingResultProps) => {

```